xbmc/pvr/epg/EpgSearchFilter.h

```diff
@@ -151,11 +156,14 @@ namespace PVR
     // PVR specific filters
     bool m_bIsRadio; /*!< True to filter radio channels only, false to tv only */
     int m_iClientID = -1; /*!< The client id */
+    int m_iChannelGroupID{-1}; /*! The channel group id */
```

Comment:
Why does this have a different initialiser to the other lines?

Reply:
coding guidelines: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#54-initialization

Reply:
Ok, well you learn something new everyday.